charts/platform-tenant/templates/rbac/cluster-role-tenant.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.rbac.enable .Values.rbac.tenantClusterRole.create -}}
+{{- if and .Values.rbac.enabled .Values.rbac.tenantClusterRole.create -}}
```

Comment:
Let's stick w/ `enabled` instead of `create` everywhere. The only alternative I think is `install` when that's appropriate.